src/components/Home/CartListProduct.jsx

```diff
@@ -1,8 +1,27 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import { addLocalStorage } from '../../services/dataLocalStorage';
 
 export default class CartListProduct extends Component {
+  addToCart = () => {
+    const { product } = this.props;
+    const {
+      title,
+      thumbnail,
+      price,
+      id,
+    } = product;
+    const newProduct = {
```

Comment:
Eu só trocaria o nome da variável, e já peço desculpas, porque eu que estava te auxiliando no momento e sugeri esse nome sem pensar 😅 então estou te sugerindo esse novo nome depois de refletir um pouco.
```suggestion
    const cartItem = {
```